jobs/JGLOBAL_WAVE_POST_BNDPNT

```diff
@@ -23,11 +20,10 @@ export DOBLL_WAV='NO'  # Bulletin post
 export DOBNDPNT_WAV='YES'  # Boundary points
 
 # Execute the Script
-${SCRgfs}/exgfs_wave_post_pnt.sh
-err=$?
-if [ ${err} -ne 0 ]; then
-  echo "FATAL ERROR: ex-script of GWES_POST failed!"
-  exit "${err}"
+${SCRgfs}/exgfs_wave_post_pnt.sh && true
+export err=$?
+if [[ ${err} -ne 0 ]]; then
+  err_chk "FATAL ERROR: ex-script of GWES_POST failed!"
```

Comment:
I don't have a preference, but elsewhere, we just `err_chk` without the `if` and message.
Whatever we choose, should be consistent.

Reply:
And perhaps GWES_POST -> JGLOBAL_WAVE_POST_BNDPNT failed?

Reply:
@aerorahul I think we should wrap it in an `if` block to make log grepping easier.  If it isn't in an if block, then `FATAL ERROR` will be written to the log.  This was something @KateFriedman-NOAA requested.  That said, it may make more sense to replace these with `err_exit` since we know `err` is not 0.  Either way, it should be standardized.

Here's my suggestion.  If we are providing a custom error message, then `err_exit` should be called within an if block.  If we are using the standard `prod_util` exit message (i.e. `FATAL ERROR, ERROR in $pgm, return code $err`), then use `err_chk`.  Thoughts?  If agreed, I'll open another issue to adopt this standard.

@JessicaMeixner-NOAA Sounds good to me.

Reply:
After discussing offline with @aerorahul, I opened https://github.com/NOAA-EMC/global-workflow/issues/3678 with a standardized approach to handle error checking.  This will be handled in a follow-on PR.

Reply:
Thanks Dave!